js/background.js

```diff
@@ -40,7 +43,19 @@ function Background() {
 
         localStorage['atb'] = 'v' + majorVersion + '-' + minorVersion;
     }
-  }
+
+	//});
```

Comment:
@bsstoner this is here so that you can just uncomment this line and comment out line 58 and test this extension by having `https://mrshu.duckduckgo.com` opened when reloading it from `chrome://extensions`.